nmctl.rst

```diff
@@ -12,7 +12,7 @@ Start with getting the binary
 
 .. code-block::
 
-  wget https://github.com/gravitl/netmaker/releases/latest/download/nmctl
+  wget -O nmctl https://github.com/gravitl/netmaker/releases/latest/download/nmctl-linux-amd64
```

Comment:
instead, just mention the link to releases download page and ask the user to download the nmctl for their OS

Reply:
Please review the requested changes again.